### ISSUE TITLE
U.I Freezing

### ISSUE THREAD

**Issue by xzackly7** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#1866]
**OS:** Windows 10

**Issue Descriptions:** The Player freezes randomly and for a random amount of time after playing a song and waiting a moment, no buttons work or anything to skip songs or thumbs up, no hotkeys, I'm using the latest update from today. The music still plays but I can't do anything on the player. This has been happening for a month or two. It hhappens even when the player is not minimized. Any time you click on anything it gives you that windows noise "bling".

**Steps to Reproduce:** I'm not really sure how to reproduce this, it just seems to happen every time I start listening to a song and then while playing a game I see that it has randomly frozen, not sure what causes it exactly.

**Comment by MarshallOfSound** [comment]
This was fixed in `4.0.2` that was released recently

**Comment by uvbeenzaned** [comment]
Not fixed. I completely uninstalled the app and reinstalled 4.0.2 and it does the same thing.

**Comment by wynnw** [comment]
I see the same thing with 4.0.3 intermittently. Using windows 10 with latest chrome installed. I'm seeing the same problem with the slack windows app interestingly enough.

**Comment by jostrander** [comment]
For the record the version of Chrome you have installed makes no difference here. GPMDP has an internal version of Chrome that is based on the Electron Framework.

**Comment by wynnw** [comment]
Seems like my problems are cured if I disable windows notifications for these two apps. Interesting...

**Comment by jasekiw** [comment]
I am running 4.0.3 and am still experiencing the freezing. This needs to be reopened. @MarshallOfSound  

I just tried disabling notifications from the app and haven't experienced freezing yet. I will update later if I run into a freeze or not

**Comment by MarshallOfSound** [comment]
> Seems like my problems are cured if I disable windows notifications for these two apps. Interesting...

This is exactly the same thing that fixed it for the other people reporting it.  I thought I had handled it but maybe not.  Just like other issues without me being able to repro it, it becomes quite tricky.   If anyone is available for a teamviewer session at some point ping me here or on Gitter and I can take a look again 👍

**Comment by jasekiw** [comment]
@MarshallOfSound I originally did not have this issue originally but then my computer messed up and I had to wipe and reinstall windows due to unrelated problems. It is interesting that the issue started after I wiped the machine. I would be willing to teamview one day. I want to make sure I can consistently cause the issue and remove it by enabling and disabling the notifications for the application before I do so though

**Comment by MarshallOfSound** [comment]
@jasekiw That is actually really interesting.  This is a quote from the last guy (who I thought I fixed it for)

> This started with a new Windows install.

Maybe this is a bug at the OS level and only new installs of a certain build are experiencing it.

**Comment by jasekiw** [comment]
@MarshallOfSound  The interesting thing is that I believe the build of the fresh install is the same as my other computer with does not experience this bug. You were mentioning in another issue that there are some optional dependencies that can help. I may go through that list and see if that machine has any of those installed. Also it has been hours since I disabled notifications for the app and there hasn't been a single freeze so it is definitely related to the notification system in windows.

**Comment by jfmcbrayer** [comment]
I see the same thing on a fresh install on Linux today.  google-play-music-desktop-player-4.0.3-1.x86_64 on Fedora 24, 64-bit.

**Comment by jfmcbrayer** [comment]
Let me amend my previous comment. It appears to only be an issue when the playback API is enabled and the remote app (Android) is connected or has previously connected.

**Comment by MarshallOfSound** [comment]
@jfmcbrayer Your issue sounds like same symptoms but different cause.  Can you ensure that you don't have a firewall blocking the PlaybackAPI port (5672)

**Comment by jfmcbrayer** [comment]
I unblocked the PlaybackAPI port in firewalld before experiencing those problems. The problems started after the client connected.

**Comment by MarshallOfSound** [comment]
@jfmcbrayer Do you have the mdns libraries installed on your distro.  `avahi`?

**Comment by beheb** [comment]
Not sure if this will help at all, but I had the same issue but the main issue for me was after a rebuild of windows 10, the major difference between versions was the anniversary update, which I had blocked until this time. I realise there are a LOT of other patches in between I had also been holding back due to this, but I am just hoping it may help. Disabling the notification popup in the player also fixed this for me but was not required before the rebuild.

**Comment by jfmcbrayer** [comment]
avahi-0.6.32, corresponding releases of avahi-gobject, avahi-libs, avahi-autoipd, avahi-glib, and avahi-ui-gtk3.

**Comment by MarshallOfSound** [comment]
My fix for this should fix it, the problem is (looking at the released version) I pushed out a built version without my change (I don't even know how).  Next update should fix

**Comment by iHydra** [comment]
I'm experiencing this issue as well even with the latest update. My laptop works fine but I recently upgraded my PC with new parts and had to reinstall Windows 10 (went from Enterprise to Education and AMD to Intel CPU). It freezes and with each time I right click the taskbar icon, it "updates" a frame (as if it is like a video frame).
